go/pkg/array/fill_test.go

```
@@ -0,0 +1,220 @@
+			slice := make([]int, size)
+			for j := 0; j < size; j++ {
+				slice[j] = rand.Intn(1000)
+			}
+		}
+	})
+
+	b.Run("Append_based", func(b *testing.B) {
+		for i := 0; i < b.N; i++ {
+			var slice []int
+			for j := 0; j < size; j++ {
+				slice = append(slice, rand.Intn(1000))
```

golangci-lint (2.2.2)

SA4010: this result of append is never used, except maybe in other appends

(staticcheck)